docs/console.html

```diff
@@ -0,0 +1,804 @@
+                Simply have the specifier at the start of the string and give the style you wish to apply as the second parameter.
+            </p>
+
+            <figure>
+                <figcaption>Styling log information</figcaption>
+                <figure>
+                    <figcaption>Example code:</figcaption>
+                    <pre class="prettyprint"><code class="language-javascript">
+console.log("%cThis will be formatted with large, blue text", "color: blue; font-size: x-large");                       
+</code></pre>
+                </figure>
+                <figure>
+                    <img src="console-files/format-string.png" alt="Formatted string" />
+                    <figcaption>Example code output.</figcaption>
+                </figure>
+            </figure>
+        </section>
+
+        <section>
+            <h3 id="measuring-how-long-something-takes">Measuring how long something takes</h3>
+            <p>
+                Time can be calculated by using the <code><a href="console-api#consoletimelabel">time()</a></code> method.
+                You must pass a string to the method to identify the time marker.
+                When you want to end the measurement call <code><a href="console-api#consoletimeendlabel">timeEnd()</a></code> and pass it the same string you passed to the initializer.
+                Once <code>timeEnd()</code> is called the label and time elapsed will be logged in the console.
+            </p>
+
+            <figure>
+                <figcaption>Example code and output for timing JavaScript execution.</figcaption>
+                <figure>
+                    <figcaption>Example Code:</figcaption>
+                    <pre class="prettyprint"><code class="language-javascript">
+console.time("Array initialize");
+    var array= new Array(1000000);
+    for (var i = array.length - 1; i >= 0; i--) {
+        array[i] = new Object();
+    };
+console.timeEnd("Array initialize");                        
+</code></pre>
+                </figure>
+                <figure>
+                    <figcaption>Output in the console:</figcaption>
+                    <img src="console-files/time-duration.png" alt="Time elapsed" />
+                </figure>
+            </figure>
+        </section>
+
+        <section>
+            <h3 id="marking-the-timeline">Marking the Timeline</h3>
+            <p>
+                The <a href="timeline">Timeline panel</a> provides a complete overview of where time is spent by the engine on a page.
+                You can add a mark to the timeline from the console with the <code><a href="console-api#consoletimestamplabel">timeStamp()</a></code>.
+                This is a simple way to correlate events in your application with other events.
+            </p>
+
+            <p class="note"><b>Note:</b> The <code>timestamp()</code> method only functions while a Timeline recording is in progress.</p>
```

Comment:
hmm... Looks like something we could find/build a checker for travis to run. Nice catch.